packages/dx-react-grid/docs/guides/editing.md

```diff
@@ -62,3 +62,13 @@ You can handle other keyboard actions. For example, the demo below shows how to
 You can prevent editing of a specific column using the [EditingState](../reference/editing-state.md) plugin's `columnExtensions` property.
```

Comment:
```suggestion
You can use the [EditingState](../reference/editing-state.md) plugin's `columnExtensions` property to prevent a specific column for being edited.
```